test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -550,6 +550,14 @@ public void startup(ICluster cluster)
         sync(() -> {
             try
             {
+                if (config.has(GOSSIP))
+                {
+                    // TODO: hacky
+                    System.setProperty("cassandra.ring_delay_ms", "5000");
```

Comment:
Did we want 15 second here (as it is in the 4.0 and trunk branches)?

Reply:
Yes indeed, well spotted.